test/tests.cpp

```diff
@@ -48,17 +48,30 @@ void log(QInterfacePtr p) { std::cout << std::endl << std::showpoint << p << std
 
 unsigned char* cl_alloc(size_t ucharCount)
```

Comment:
Still seems weird to me to have these calls in the test/tests.cpp code, since don't we have the same functionality elsewhere?